src/main/java/com/mkopec/ticketbookingapp/domain/Movie.java

```diff
@@ -0,0 +1,26 @@
+package com.mkopec.ticketbookingapp.domain;
+
+import lombok.Data;
+
+import javax.persistence.Basic;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+import static javax.persistence.GenerationType.IDENTITY;
+
+@Entity
+@Data
+public class Movie {
+
+    @Id
+    @GeneratedValue(strategy = IDENTITY)
+    private Long id;
+
+    private String title;
+
+    private String director;
+
+    @Basic
+    private java.time.LocalTime length;
```

Comment:
import? i po co ten `@Basic` ?

Reply:
Nie wiem o który import tutaj chodzi, prawdopodobnie o import static. Słyszałem, że dobrą praktyką jest import statyczny tych stałych (zwłaszcza gdy w jednej adnotacji może ich występować więcej). A adnotacja @Basic to chyba pozostałość po wcześniejszych wersjach springa, ogólnie wzorowałem się na: https://www.baeldung.com/hibernate-date-time a tam pola oznaczone są tą adnotacją. W springu 5 działa dobrze już bez tej adnotacji.

Reply:
Chodziło mi o to, że tą klasę `LocalTime` można zaimportować i nie mieć kropeczek wtedy (java.time można się pozbyć)

Reply:
Racja, mój błąd